apps/frontend/src/features/battle/components/setup/BattleSetupContainer.tsx

```
@@ -32,11 +32,11 @@ export const BattleSetupContainer = ({
   onStartBattle,
   onCopyLink,
 }: BattleSetupContainerProps) => (
-  <div css={mainContainerStyle}>
-    <section css={leftSectionStyle}>
+  <div css={mainContainerStyle} role="region" aria-label="배틀 대기실">
+    <section css={leftSectionStyle} aria-label="참가자 목록">
```

Biome (2.3.13)

[error] The elements with this role can be changed to the following elements:
<section>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)